tests/src/rules/no-restricted-paths.js

```diff
@@ -255,7 +322,143 @@ ruleTester.run('no-restricted-paths', rule, {
         } ],
       } ],
       errors: [ {
-        message: 'Restricted path exceptions must be glob patterns when`from` is a glob pattern',
+        message: 'Restricted path exceptions must be glob patterns when `from` contains glob patterns',
+        line: 1,
+        column: 15,
+      } ],
+    }),
+
+    // support of arrays for from and target
+    // array with single element
+    test({
+      code: 'import b from "../server/b.js"',
+      filename: testFilePath('./restricted-paths/client/a.js'),
+      options: [ {
+        zones: [ {
+          target: ['./tests/files/restricted-paths/client'],
+          from: './tests/files/restricted-paths/server',
+        } ],
+      } ],
```

Comment:
```suggestion
      options: [
        {
          zones: [
            {
              target: ['./tests/files/restricted-paths/client'],
              from: './tests/files/restricted-paths/server',
            }
          ],
        }
      ],
```
and similar formatting in all the test cases

Reply:
@ljharb looks good 👍🏽 https://github.com/import-js/eslint-plugin-import/pull/2466/commits/76883fd77dc5bf42e2cee1c3af30243412f5fbb6